metadata_layer.py

```diff
@@ -590,7 +590,7 @@ def set_default_rights_uri(self, data_source):
         for format in self.formats:
             if format.link:
                 link = format.link
-                if self.rights_uri == None and link.rel == Hyperlink.OPEN_ACCESS_DOWNLOAD:
+                if self.rights_uri in (None, RightsStatus.UNKNOWN) and link.rel == Hyperlink.OPEN_ACCESS_DOWNLOAD:
```

Comment:
I think if it was unknown before, and then you see an open-access link, it becomes generic open access. I can't think of any counterexamples.